.Lib9c.Tests/Action/BattleArena4Test.cs

```diff
@@ -836,5 +836,139 @@ public void Execute_CoolDownBlockException()
                 Random = new TestRandom(),
             }));
         }
+
+        [Fact]
+        public void Execute_v100291()
+        {
+            var keys = new List<string>
+            {
+                nameof(SkillActionBuffSheet),
+                nameof(ActionBuffSheet),
+                nameof(StatBuffSheet),
+            };
+            foreach (var (key, value) in _sheets)
+            {
+                if (keys.Contains(key))
+                {
+                    _state = _state.SetState(Addresses.TableSheet.Derive(key), null!);
+                }
+            }
+
+            int championshipId = 1;
+            int round = 1;
+            Assert.True(_state.GetSheet<ArenaSheet>().TryGetValue(
+                championshipId,
+                out var row));
+
+            Assert.True(row.TryGetRound(1, out var roundData));
```

Comment:
Did you intended this?

```suggestion
            Assert.True(row.TryGetRound(round, out var roundData));
```